monitoring-support/src/main/java/org/terracotta/monitoring/PlatformMonitoringConstants.java

```diff
@@ -33,6 +33,10 @@
    */
   public static final String PLATFORM_ROOT_NAME = "platform";
   /**
+   * The name of the node in the tree which is the parent to all nodes representing connected servers.
+   */
+  public static final String SERVER_ROOT_NAME = "servers";
+  /**
```

Comment:
I think you would need that also to be consistent with the other constants:

`public static final String[] SERVERS_PATH = new String[]{PLATFORM_ROOT_NAME, SERVERS_ROOT_NAME};`